mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,14 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Mercenaries:
+		Name: Mercenaries
+		InternalName: mercenaries
+		Selectable: false
+	Faction@Smugglers:
+		Name: Smugglers
+		InternalName: smuggler
```

Comment:
Internal name `smuggler` (singular) vs `mercinaries` (plural) seems a bit odd.  Is this correct?